ag/ag.proto

```diff
@@ -73,14 +74,15 @@ message Course {
     string tag                     = 6;
     string provider                = 7;
     uint64 organizationID          = 8;
-    string organizationPath        = 9;  // The organization's SCM name, e.g. uis-dat520-2020.
-    uint32 slipDays                = 10;
-    string dockerfile              = 11;
-    Enrollment.UserStatus enrolled = 12 [(go.field) = { tags: 'gorm:"-"' }];
-
-    repeated Enrollment enrollments = 13;
-    repeated Assignment assignments = 14;
-    repeated Group groups           = 15;
+    string organizationPath        = 9;   // The organization's SCM name, e.g. uis-dat520-2020.
+    uint64 installationID          = 10;  // Each organization has a unique GitHub App installation ID
```

Comment:
Suggest to use `int64` instead to avoid casting later. See other comment.

Reply:
Done.